src/popup/components/emoji-picker.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react'
+import { useState } from 'react'
+// import "emoji-mart/css/emoji-mart.css";
```

Comment:
Some bundler/compilers like webpack have the ability to import css. We're not using webpack, but are using our own [build script](https://github.com/morehumaninternet/roar-extension/blob/main/package.json#L6) that first clears the build folder, then compiles using typescript, then bundles using our [bundle.sh](https://github.com/morehumaninternet/roar-extension/blob/main/bundle.sh#L21) script that uses [browserify](https://github.com/browserify/browserify) and bash. 

So to add the css file you would, in that bundle.sh script, write

```bash
cp node_modules/emoji-mart/css/emoji-mart.css ./$bundled_dir
```

Then, when you start/restart the build, you should see a file get copied in at `bundled/emoji-mart.css`

Finally, you can add the stylesheet to the [popup's HTML page](https://github.com/morehumaninternet/roar-extension/blob/main/html/popup.html#L7) just as we've done for `Draft.css`.

Hope this helps! I'll document this better in the Readme so others know how to add custom stylesheets.

Reply:
Great help!